pkg/scheduler/scheduler.go

```diff
@@ -422,7 +429,11 @@ func (sched *Scheduler) Run(ctx context.Context) {
 func (sched *Scheduler) recordSchedulingFailure(podInfo *framework.PodInfo, err error, reason string, message string) {
 	sched.Error(podInfo, err)
 	pod := podInfo.Pod
-	sched.Recorder.Eventf(pod, nil, v1.EventTypeWarning, "FailedScheduling", "Scheduling", message)
```

Comment:
I expected the fallback logic here: https://github.com/kubernetes/kubernetes/pull/83692/files#r339204676

not in if/else blocks at event record time